docs/building-deployer-helm.md

```diff
@@ -366,53 +379,27 @@ If you following the recommendation of creating a wrapper
 chart `wordpress-mp`, the two properties should be
 `wordpress.image.repository` and
 `wordpress.mariadb.image.repository`. The schema
-file should then look something like this:
+file should then look like this:
 
 ```yaml
-# THIS DOES NOT WORK. READ FURTHER!
-properties:
-  wordpress.image.repository:
-    type: string
-    default: bitnami/wordpress   # This is needed.
-    x-google-marketplace:        # This annotation is how the system
-      type: IMAGE                # knows to pass the image name.
-  wordpress.mariadb.image.repository:
-    type: string
-    default: bitnami/mariadb
-    x-google-marketplace:
-      type: IMAGE
-```
-
-There is one last problem: the images in these charts are specified
-by 3 separate components. Marketplace passes the full image name
-to a single property (something like
-`marketplace.gcr.io/mariadb:10.1.33`). We can use the splitting
-feature of the `IMAGE` property type.
-
-```yaml
-properties:
-  wordpressImage:
-    type: string
-    default: gcr.io/your-company/wordpress:4.9.6
-    x-google-marketplace:
-      type: IMAGE
-      image:
-        generatedProperties:
-          splitToRegistryRepoTag:
-            registry: wordpress.image.registry
-            repo: wordpress.image.repository
-            tag: wordpress.image.tag
-  mariadbImage:
-    type: string
-    default: gcr.io/your-company/mariadb:10.1.133
-    x-google-marketplace:
-      type: IMAGE
-      image:
-        generatedProperties:
-          splitToRegistryRepoTag:
-            registry: wordpress.mariadb.image.registry
-            repo: wordpress.mariadb.image.repository
-            tag: wordpress.mariadb.image.tag
+ # Under the x-google-marketplace parent attirbute
+  images:
+    '':
```

Comment:
We no longer require this __primary image__ concept actually. So we can give it a real name, `wordpress`.
```suggestion
    wordpress:
```

Reply:
Done!